Add case for the aarch64 architecture

Taken from https://bugs.launchpad.net/ubuntu/+source/util-linux/+bug/1061609

Code written by Wookey.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/fdisk/fdiskbsdlabel.h
===================================================================
--- a/fdisk/fdiskbsdlabel.h
+++ b/fdisk/fdiskbsdlabel.h
@@ -48,7 +48,8 @@
 
 #if defined (i386) || defined (__sparc__) || defined (__arm__) || \
     defined (__mips__) || defined (__s390__) || defined (__sh__) || \
-    defined(__x86_64__) || defined (__avr32__) || defined(__cris__)
+    defined(__x86_64__) || defined (__avr32__) || defined(__cris__) || \
+    defined(__aarch64__)
 #define BSD_LABELSECTOR   1
 #define BSD_LABELOFFSET   0
 #elif defined (__alpha__) || defined (__powerpc__) || defined (__ia64__) || defined (__hppa__)
